src/convertToObject.js

```diff
@@ -1,12 +1,25 @@
 'use strict';
 
 /**
- * @param {string} sourceString
+ * Convert a CSS string into an object
  *
+ * @param {string} sourceString
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0)
+    .reduce((cssObject, style) => {
+      const [key, value] = style.split(':').map((part) => part.trim());
```

Comment:
The current implementation of `style.split(':')` assumes that there is only one colon in each style declaration. If a style value contains a colon (e.g., `background: url(http://example.com)`), this will not work as expected. Consider using a more robust method to handle such cases, although handling multiple colons is not required by the task.